src/format/parse.rs

```diff
@@ -488,7 +488,7 @@ where
                     }
 
                     &RFC2822 => try_consume!(parse_rfc2822(parsed, s)),
-                    &RFC3339 => try_consume!(parse_rfc3339(parsed, s)),
+                    &RFC3339 => try_consume!(parse_rfc3339_relaxed(parsed, s)),
```

Comment:
What's the motivation for this?

Reply:
This PR started after the discussion in https://github.com/chronotope/chrono/pull/1083#discussion_r1209047686.

The `RFC3339` formatting item is used for the `%+` specifier. It has the following documentation:

  > Same as `%Y-%m-%dT%H:%M:%S%.f%:z`, i.e. 0, 3, 6 or 9 fractional digits for seconds and colons in the time zone offset.
  >
  > This format also supports having a `Z` or `UTC` in place of `%:z`. They are equivalent to `+00:00`.
  >
  > Note that all `T`, `Z`, and `UTC` are parsed case-insensitively.
  >
  > The typical `strftime` implementations have different (and locale-dependent) formats for this specifier. While Chrono's format for `%+` is far more stable, it is best to avoid this specifier if you want to control the exact output.

Nowhere does it say this is a strict RFC 3339 parser, it pretty much says the opposite.
And I think this definition has its uses, as it allows parsing the debug format of `DateTime<Utc>`.

Reply:
Okay.